packages/ketcher-polymer-editor-react/src/components/App/App.tsx

```diff
@@ -20,6 +20,7 @@ import { useAppDispatch, useAppSelector } from 'hooks'
 import { selectEditorIsReady, fetchInitData } from 'state/common'
 import { MonomerLibrary } from 'components/monomerLibrary'
 import styled from '@emotion/styled'
+import { AddToCanvas } from 'components/addToCanvas/addToCanvas'
```

Comment:
it's better to re-export addToCanvas from index.ts to avoid this /addToCanvas/addToCanvas